src/js/components/Drop/DropContainer.js

```diff
@@ -243,13 +243,15 @@ const DropContainer = forwardRef(
         scrollParents.forEach(scrollParent =>
           scrollParent.addEventListener('scroll', place),
         );
+        window.addEventListener('scroll', place, true);
```

Comment:
How is this different from ensuring that `document` is in the `scrollParents`?

Reply:
Good comment. I've adjusted findScrollParents to add `document` as a scrollParent and removed it from here. Previously, it was only being added to the list if there were no other scrollable regions on the page, but it seems that we always want to have it included.